### PR TITLE
adding dai warning for erc20 token vs xdai native token

### PR DESCRIPTION
**Pull request by brossetti1**
in reference [to this discussion](https://discordapp.com/channels/684227450204323876/744945190401474710/757340078354727044)

I opened an [issue here](https://github.com/raid-guild/xdai-omnibridge/issues/82)

I chose not to make this a seperate component because it is just an alert, and its not very complicated. let me know if you need anything to change here, thanks.

### warning in confirm modal:
Screen Shot 2020-09-20 at 3 26 21 PM

### warning when dai is selected:
Screen Shot 2020-09-20 at 3 46 01 PM

### CODE REVIEW

File: packages/react-app/src/components/ConfirmTransferModal.jsx
Comment (dan13ram) [inline packages/react-app/src/components/ConfirmTransferModal.jsx:39]: In either cases, should we do a hard-coded address check instead of symbol check?

**Review by dan13ram — CHANGES_REQUESTED** [review]
Looks great @brossetti1 
Please add this to the responsive mobile screen as well and we can merge this :D

File: packages/react-app/src/components/BridgeTokens.jsx
Comment (dan13ram) [inline packages/react-app/src/components/BridgeTokens.jsx:23]: we can actually move this alert to a separate component and reuse it

File: packages/react-app/src/components/BridgeTokens.jsx
Comment (dan13ram) [inline packages/react-app/src/components/BridgeTokens.jsx:26]: Lets wait for xDai team to approve the text as well.

**Comment by brossetti1** [comment]
@dan13ram I took another look at this one, split the `DaiWarning` into its own component, and added address checking. One of the scenarios that popped up from that is I had to add logic for which network the user is connected to. since the [content of the message](https://discordapp.com/channels/684227450204323876/744945190401474710/757836994586869760) from the xdai team is:

```
Bridges DAI on Ethereum to DAI on xDai, DOES NOT mint native xDai. If you want native xDai, use the <a href=“https://dai-bridge.poa.network/”>ExDai Bridge</a>.
``` 

I opted to restrict the `<DaiWarning /> to dai fromToken's on ethereum based networks` as the message does not apply going from xdai to dai.

I added the dai address constant in the DaiWarning component as I didn't want to clutter the constants only to store Dai addresses. Let me know what you think or if you want changes here.

### Selection process on large screen:
Screen Shot 2020-09-22 at 8 03 47 PM
Screen Shot 2020-09-22 at 8 05 12 PM

### Selection process on small screen:
Screen Shot 2020-09-22 at 8 11 51 PM
Screen Shot 2020-09-22 at 8 12 12 PM

File: packages/react-app/src/components/DaiWarning.jsx
Comment (brossetti1) [inline packages/react-app/src/components/DaiWarning.jsx:11]: a little unconventional but I dont see this information being used outside this component.

File: packages/react-app/src/components/DaiWarning.jsx
Comment (brossetti1) [inline packages/react-app/src/components/DaiWarning.jsx:18]: this is also a little unconventional and it is exported, but I only see this being used WITH this component so I put it here.

Comment (brossetti1) [inline packages/react-app/src/components/DaiWarning.jsx:18]: this is also a little unconventional and it is exported, but I only see this being used WITH this component so I put it here.

File: packages/react-app/src/components/DaiWarning.jsx
Comment (brossetti1) [inline packages/react-app/src/components/DaiWarning.jsx:11]: a little unconventional but I dont see this information being used outside this component.

File: packages/react-app/src/lib/constants.jsx
Comment (brossetti1) [inline packages/react-app/src/lib/constants.jsx:7]: ¯\_(ツ)_/¯

Comment (scottrepreneur) [reply packages/react-app/src/lib/constants.jsx:7]: got an extra e in ETHEREUM_NETWORKS

Comment (dan13ram) [reply packages/react-app/src/lib/constants.jsx:7]: Hey instead of creating this new variable there's an existing method called isxDaiChain in `lib/helpers` which helps you do the same thing.

File: packages/react-app/src/components/DaiWarning.jsx
Comment (dan13ram) [inline packages/react-app/src/components/DaiWarning.jsx:32]: It should be `xDai Bridge` not `ExDai`.
Also you don't need to set `rel="..."` when you are setting isExternal.

File: packages/react-app/src/components/BridgeTokens.jsx
Comment (dan13ram) [inline packages/react-app/src/components/BridgeTokens.jsx:3]: This is causing lint errors, please remove it. Else it won't build.

File: packages/react-app/src/components/DaiWarning.jsx
Comment (dan13ram) [inline packages/react-app/src/components/DaiWarning.jsx:24]: Can you set a minW for this icon, it becomes too small for smaller screens

**Review by dan13ram — CHANGES_REQUESTED** [review]
Great stuff man! Just some small changes before we can merge it :)

**Comment by brossetti1** [comment]
@dan13ram nice catch, fixed the remaining feedback. heres a shot of the small screen alert icon

Screen Shot 2020-09-23 at 8 03 20 AM

**Review by dan13ram — APPROVED** [review]

**Review by scottrepreneur — APPROVED** [review]
great work

**Pull request merged**
